docs/docs/the_aztec_network/index.md

```diff
@@ -1,62 +1,123 @@
+
+## Prerequisites
+
+Minimum hardware requirements:
+
+- 2 core / 4 vCPU
+- 8 GB RAM
+- 10 GB SDD
+- 25 Mbps network connection
+
+Please note that these requirements are subject to change as the network throughput increases.
+
+Along with the above minimum hardware requirements, it is assumed that the user has access to a performant ethereum RPC endpoint. Furthermore, this guide expects the user to be using a "standard" Linux distribution like Debian / Ubuntu when following along with the steps.
+
+Overview
+
+1. Install Docker
+2. Install aztec
+3. Configure the node
+4. Start the node!
+
+## Install and set up Docker
+
+Please ensure that docker is installed. If not, here is a convenient way to install it.
+This uses the script at
+[https://get.docker.com/](https://get.docker.com/) to install the
+latest stable release of Docker on Linux:
+
+```console
+curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+```
+
+Afterwards, the currently logged in user must be added to the docker group, so sudo is not needed to invoke docker.
+```console
+sudo groupadd docker
+sudo usermod -aG docker $USER
+newgrp docker
+# Invoke docker without sudo to test if the above changes have been applied successfully
+docker run hello-world
+```
+
+## Install Aztec
+
+Run these commands to grab the aztec stack and add them to path.
+
+```console
+bash -i <(curl -s https://install.aztec.network)
+# Users should check that it is installed by using
+ls ~/.aztec/bin
+# aztec, aztec-up aztec-nargo and aztec-wallet should show up here
+echo 'export PATH="$HOME/.aztec/bin:$PATH"' >> ~/.bashrc
+source ~/.bashrc
+```
+
+The next step is to install the correct version of aztec that is running on testnet. The correct version of the aztec network currently is `1.2.0`. If that is not the version that was installed in the previous step, please run this to ensure the correct version is installed.
+
+```console
+aztec-up 1.2.0
+```
+
+The user can confirm that they are running on the correct version with:
+
+```console
+aztec --version
+```
+
+## Configure the node
+
+Now that the correct version of the node is installed, it needs to be configured correctly. Let's start by making a new directory where the node data can be stored.
+
+```console
+mkdir aztec-node && cd ./aztec-node
+```
+
+Next, set some required configuration options. This guide will use `custom_named` environment variables (e.g. `AZTEC_NODE_P2P_IP`) but please not that this is not necessary; you can pass valued directly into the command without specifying them as environment variables. The external IP address of the node, and the L1 RPC endpoints must be defined when starting a node. Also, configuration defining the network version should be set, as this will let the node define the other required protocol variables, like rollup address, registry address etc. Please note that the specified RPC endpoints must support high throughput, otherwise the node will suffer degraded performance.
```

Comment:
```suggestion
Next, set some required configuration options. This guide will use `custom_named` environment variables (e.g. `AZTEC_NODE_P2P_IP`) but this is not necessary; you can pass valued directly into the command without specifying them as environment variables. The external IP address of the node, and the L1 RPC endpoints must be defined when starting a node. Also, configuration defining the network version should be set, as this will let the node define the other required protocol variables, like rollup address, registry address etc. Please note that the specified RPC endpoints must support high throughput, otherwise the node will suffer degraded performance.
```